adhocteam/Head-Start-TTADP
# [TTAHUB-311] Add admin page (& associated policies) for managing feature flags

## Description of change
Implement a basic "feature flag" system, including giving the admin users the ability to edit which users have feature flag capabilities.

- Add a column to Users (like roles) for the flags
- Use policies/Users to check access
- Admins still have blanket access
- On the Admin section, features can be removed from the user administration screen, users can be filtered by feature, and I've also added a low-functionality page to simply list active feature flags.  
- Create a component ```<FeatureFlag />``` that will conditionally hide it's children something based on user/selected flag.

There may be the ability to use cloud.gov environmental variables to activate/deactivate feature flags, however in my initial testing I failed to inject them into the front end build step. More research may yield a working method.

## How to test
Running the migrations adds a single feature flag, 'grantee_record_page' to the db. You should be able to add and remove it from users locally. The CI tests verify that the policies and components work as expected.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-311

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
